src/box/flightrec.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#include "flightrec.h"
+
+#include "say.h"
+#include "trivia/config.h"
+
+#if defined(ENABLE_FLIGHT_RECORDER)
+# error unimplemented
+#endif
+
+void
+flightrec_init(const char *fr_dirname, size_t logs_size,
+	       size_t logs_max_msg_size, int logs_log_level)
+{
+	(void)fr_dirname;
+	(void)logs_size;
+	(void)logs_max_msg_size;
+	(void)logs_log_level;
+	say_error("Flight recorder is not available in this build");
+}
+
+void
+flightrec_destroy(void)
+{
+}
+
+int
+flightrec_check_log(int64_t logs_size, int64_t logs_max_msg_size,
+		    int logs_log_level)
+{
+	(void)logs_size;
+	(void)logs_max_msg_size;
+	(void)logs_log_level;
+	return false;
+}
+
+int
+flightrec_log(int level, const char *filename, int line,
+	      const char *error, const char *format, va_list ap)
+{
+	(void)level;
+	(void)filename;
+	(void)line;
+	(void)error;
+	(void)format;
+	(void)ap;
+	return 0;
+}
```

Comment:
Nit: All these functions could be made static inline without introducing any dependencies.

Reply:
Ok, removed box/flightrec.c.